lib/matplotlib/patches.py

```diff
@@ -1081,7 +1095,7 @@ def set_values(self, values):
         ----------
         values : 1D array-like
         """
-        self.set_data(values, edges=None)
+        self.set_data(values, edges=None, baseline=None)
```

Comment:
I more and more come to the conclusion that `set_data` is sufficient and we don't need separate getters/setters for each attibute. Modifying these values explicitly after creation is rare, and `set_data(baseline=3)` / `get_data().basline` is not much worse than `set_baseline(3)` / `get_baseline()`. Let's keep the API footprint small.

Reply:
I agree with this modulo that I think getting/setting values is going to be much more common than the other 2. But I guess `set_data(values)` will still work just fine.